server/src/test/kotlin/org/itmodreamteam/myrest/server/service/table/EmployeeServiceMock.kt

```diff
@@ -0,0 +1,59 @@
+package org.itmodreamteam.myrest.server.service.table
+
+import org.itmodreamteam.myrest.server.model.restaurant.Employee
+import org.itmodreamteam.myrest.server.model.restaurant.Manager
+import org.itmodreamteam.myrest.server.model.restaurant.Waiter
+import org.itmodreamteam.myrest.server.service.employee.EmployeeService
+import org.itmodreamteam.myrest.server.service.restaurant.RestaurantService
+import org.itmodreamteam.myrest.server.service.user.UserService
+import org.itmodreamteam.myrest.shared.restaurant.*
+import org.springframework.stereotype.Service
+
+@Service
+class EmployeeServiceMock(
+    private val restaurantService: RestaurantService,
+    private val userService: UserService,
+) : EmployeeService {
+
+    override fun toEmployeeInfo(employee: Employee): EmployeeInfo {
```

Comment:
Yes, this is what I wrote earlier. Converter methods should not be in services.
- Code duplication
- Possible cycle dependencies for services